proof/archive.go

```diff
@@ -376,6 +477,7 @@ func (f *FileArchiver) FetchProofs(_ context.Context,
 			Locator: Locator{
 				AssetID:   &id,
 				ScriptKey: *scriptKey,
+				OutPoint:  outPoint,
```

Comment:
This won't be the full outpoint since we're only storing the truncated version. 

Instead, we'll need to decode enough of the proof file to be able to extract the outpoint.

Reply:
Unfortunately we need to decode the full proof here. But the `FetchProofs` method is only used to fetch all proofs of an asset when a re-org happens, so we should hit this code path only very seldomly.